experimental/traffic-portal/src/app/shared/generic-table/generic-table.component.html

```diff
@@ -49,7 +50,7 @@
 	<ul>
 		<li *ngFor="let item of contextMenuItems" role="menuitem">
 			<ng-container *ngIf="!isAction(item)">
-				<a *ngIf="!isDisabled(item)" [href]="href(item)" [target]="item.newTab ? '_blank' : '_self'">{{item.name}}</a>
+				<a *ngIf="!isDisabled(item)" [routerLink]="href(item)" [href]="href(item)" [target]="item.newTab ? '_blank' : '_self'">{{item.name}}</a>
```

Comment:
`href` and `routerLink` are mutually exclusive. I'm not sure which binding takes precedence, but they're both binding to the `href` attribute of the underlying anchor.